controller/ship/fleetController.go

```diff
@@ -143,3 +140,96 @@ func GetFleetsOnPlanet (w http.ResponseWriter, r *http.Request){
 	
 	utils.SendJsonResponse(w, 200,shipManager.GetFleetsOnPlanet(player,planet));
 }
+
+func AssignMultipleShipsToFleet (w http.ResponseWriter, r *http.Request){
+    /*
+     * Assign mutliple ship to a fleet by theire id given in the body ( json) {"data-ships" : [id,id,id]}
+     */
+    
+    player := context.Get(r, "player").(*model.Player)
+    
+    idFleet, _ := strconv.ParseUint(mux.Vars(r)["fleetId"], 10, 16)
+    fleet := shipManager.GetFleet(uint16(idFleet));
+    data := utils.DecodeJsonRequest(r)["data-ships"].([]interface{});
+    
+    var ships []*model.Ship
+    
+    if (player.Id != fleet.Player.Id) { // the player does not own the planet
+		panic(exception.NewHttpException(http.StatusForbidden, "", nil));
+	}
+    
+    for i := range data {
+        idShip := data[i].(float64)
+        ship := shipManager.GetShip(uint16(idShip));
```

Comment:
Il faudrait faire une requête qui utilise un WHERE IN avec le tableau d'ID en paramètres, là potentiellement si tu ajoutes 100 vaisseaux d'un coup tu fais 100 requêtes, en termes de perf c'est pas terrible ! un WHERE IN permettra de conserver une seule requête quoi qu'il arrive.